share/doc/INSTALL.md

```diff
@@ -46,18 +46,20 @@ To install the full package:
 
 ### Resolving the 'not found' dependency errors
 
-Assuming all the [dependencies](#dependency-installation) have been correctly installed, one or more might still appear 'not found' upon configuring with `cmake` command (see above). In this case you will need to find out the 'non-standard' location for each missed out dependency (most often a shared or dynamically loaded library, e.g. libgromacs.so.\*,  libhdf5.so.\* etc). 
+Assuming all the [dependencies](#dependency-installation) have been correctly installed, one or more might still appear 'not found' upon configuring with `cmake` command (see above). In this case you will need to find out the 'non-standard' location for each missed out dependency (most often a shared or dynamically loaded library, e.g. `libgromacs.so.*`,  `libhdf5.so.*` etc). 
```

Comment:
You can remove the out, in this section which appears at the end of the second sentence. 

"missed out dependency"